packages/components/src/components/SquiggleChart.tsx

```diff
@@ -8,20 +8,23 @@ import {
   defaultBindings,
   defaultEnvironment,
 } from "@quri/squiggle-lang";
-import { FunctionChartSettings } from "./FunctionChart";
 import { useSquiggle } from "../lib/hooks";
 import { SquiggleErrorAlert } from "./SquiggleErrorAlert";
 import { SquiggleItem } from "./SquiggleItem";
 
 export interface SquiggleChartProps {
   /** The input string for squiggle */
-  squiggleString?: string;
+  code?: string;
   /** If the output requires monte carlo sampling, the amount of samples */
   sampleCount?: number;
   /** The amount of points returned to draw the distribution */
   environment?: environment;
-  /** If the result is a function, where the function starts, ends and the amount of stops */
-  chartSettings?: FunctionChartSettings;
+  /** If the result is a function, where the function domain starts */
+  diagramStart?: number;
+  /** If the result is a function, where the function domain ends */
+  diagramStop?: number;
+  /** If the result is a function, the amount of stops sampled */
+  diagramCount?: number;
```

Comment:
My current impression is that we should nest props more to keep them manageable in the future, while this goes in the opposite direction.

Reply:
Oh, it was for consistency with SqugigleEditor! No problem then, good idea.